apps/src/templates/projects/FeaturedProjectsTable.story.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import FeaturedProjectsTable from './FeaturedProjectsTable';
+
+export default storybook => {
+  storybook
+    .storiesOf('FeaturedProjectsTable', module)
+    .addStoryTable([
+      {
+        name: 'Featured Project Table - current',
+        description: 'Table of currently featured projects projects',
```

Comment:
I know these are a work in progress, but others who use storybook might not know this. Could we update the description to something more reflective of the current state? like "Placeholder for work in progress featured progress table"?